src/style/_variables.scss

```diff
@@ -1,269 +1,275 @@
 // Base Colours
-$black: #000;
-$white: #fff;
-$gray-50: lighten($black, 92%); //ebebeb
-$gray-200: lighten($black, 62.75%); // #a0a0a0
-$gray-300: lighten($black, 56.5%); // #909090
-$gray-400: lighten($black, 50%); // #808080
-$gray-500: lighten($black, 43.75%); // #707070
-$gray-600: lighten($black, 37.5%); // #606060
-$gray-650: lighten($black, 33.3%); // #555555
-$gray-700: lighten($black, 31.25%); // #505050
-$gray-800: lighten($black, 25%); // #404040
-$gray-900: lighten($black, 18.75%); // #303030
-
-$cod-gray: #1b1b1b;
-$bright-gray: #3b4151;
-$mako-gray: #41444e;
-$waterloo-gray: #7d8492;
-$alto-gray: #d9d9d9;
-$mercury-gray: #e4e4e4;
-$concrete-gray: #e8e8e8;
-$alabaster: #f7f7f7;
-$apple-green: #62a03f;
-$green-haze: #009d77;
-$japanese-laurel: #008000;
-$persian-green: #00a0a7;
-$geyser-blue: #d8dde7;
-$dodger-blue: #1391ff;
-$endeavour-blue: #005dae;
-$scampi-purple: #55a;
-$electric-violet: #7300e5;
-$persian-red: #cf3030;
-$mango-tango: #e97500;
+
+$color-black: #000;
+$color-white: #fff;
+$color-cod-gray: #1b1b1b;
+$color-mine-shaft: #333333;
+$color-waterloo-gray: #7d8492;
+$color-bright-silver: #c8c8c8;
+$color-silver: #cbcbcb;
+$color-alto-gray: #d7d7d7;
+$color-iron-gray: #d9d9db;
+$color-mercury-gray: #e4e4e4;
+$color-wild-sand: #f5f5f5;
+$color-alabaster: #fafafa;
+$color-mantis-green: #57c534;
+$color-apple-green: #62a03f;
+$color-vida-loca: #49a32b;
+$color-green-haze: #009d77;
+$color-japanese-laurel: #008000;
+$color-persian-green: #00a0a7;
+$color-havelock-blue: #4990e2;
+$color-dodger-blue: #1391ff;
+$color-endeavour-blue: #005dae;
+$color-governor-bay: #4234c5;
+$color-electric-violet: #7300e5;
+$color-berry-red: #990000;
+$color-guardsman-red: #bb0000;
+$color-persian-red: #cf3030;
+$color-coral: #ff6060;
+$color-mango-tango: #e97500;
+$color-soft-peach: #f9f2f4;
+
+$color-gray-50: lighten($color-black, 92%); //ebebeb
+$color-gray-150: lighten($color-black, 70%); // #b3b3b3
+$color-gray-200: lighten($color-black, 62.75%); // #a0a0a0
+$color-gray-300: lighten($color-black, 56.5%); // #909090
+$color-gray-400: lighten($color-black, 50%); // #808080
+$color-gray-500: lighten($color-black, 43.75%); // #707070
+$color-gray-650: lighten($color-black, 33.3%); // #555555
+$color-gray-800: lighten($color-black, 25%); // #404040
 
 // Theme
 
-$color-primary: #89bf04 !default;
-$color-secondary: #9012fe !default;
-$color-info: #4990e2 !default;
-$color-warning: #ff6060 !default;
-$color-danger: #f00 !default;
+$color-primary: $color-vida-loca !default;
+$color-secondary: $color-mine-shaft !default;
+$color-info: $color-havelock-blue !default;
+$color-warning: $color-coral !default;
+$color-danger: $color-guardsman-red !default;
+
+$color-primary--hover: $color-mantis-green !default;
+$color-secondary--hover: $color-gray-500 !default;
+
+$color-type--default: $color-mine-shaft !default;
```

Comment:
**Question**: I'm not seeing how `default` is being used as a modifier here (why not just `color-type`?) -- would appreciate some illumination

Reply:
I was unsure when I was putting it in there myself! 

Just for the reason that, although the majority of the text is that color, not all of the text is. Hence the why I snuck "default" in the name. 

But thinking about it now... Yeah, "default" probably shouldn't be classed as a modifier. Even if it's a variable.